workstations.yml

```diff
@@ -1,12 +1,11 @@
 ---
-- hosts: clients:mediaboxen
+- hosts: workstations:mediaboxen
   roles:
-    - network
     - xbps-repoconf
     - core-daemons
     - core-programs
     - sshd
-    - client
     - netlogon
     - cups-client
     - netdata
+    - workstation
```

Comment:
This is up for debate, in theory you could argue that netlogon is a dependency of the workstation role.